res/css/views/rooms/_EventBubbleTile.pcss

```diff
@@ -475,28 +475,25 @@ limitations under the License.
             grid-template:
                 "reply reply" auto
                 "shield body" auto
-                "shield link" auto
```

Comment:
The `"link"` is the "Click here to resend key requests" link that was on every message, and has been removed. The `"shield"` is another part of the `"shield"` region that begins in the previous row of the grid layout, and used to extend into the link row, but the link row no longer exists because the link is gone.